src/elements/checkbox/checkbox-group/checkbox-group.ts

```diff
@@ -60,10 +60,7 @@ class SbbCheckboxGroupElement extends SbbDisabledMixin(LitElement) {
 
   public override connectedCallback(): void {
     super.connectedCallback();
-    this.toggleAttribute(
-      'data-has-panel',
-      !!this.querySelector?.('sbb-selection-expansion-panel, sbb-checkbox-panel'),
-    );
+    this.toggleAttribute('data-has-panel', !!this.querySelector?.('sbb-checkbox-panel'));
```

Comment:
Is the `selection-expansion-panel` not needed anymore? same for radio-group

Reply:
I think not, as you never use a selection expansion panel without a containing checkbox or radio panel. with queryselector you will also find the underlaying checkbox panel or radio panel.

Reply:
do you agree?